compiler/aarch64/codegen/OMRInstOpCodeEnum.hpp

```diff
@@ -477,6 +477,13 @@
 		fmaxd,                                                  	/* 0x1E604800	FMAX      	 */
 		fmins,                                                  	/* 0x1E205800	FMIN      	 */
 		fmind,                                                  	/* 0x1E605800	FMIN      	 */
+	/* Vector Move (2 source) */
```

Comment:
I would like this comment to be changed.
How about this, merging the sections of `vorr2d` and `vadd*` instructions?

```
/* Vector Data-processing (2 source) */
```